apps/theme/components/ColorPicker/ColorPicker.tsx

```diff
@@ -61,7 +61,7 @@ export const ColorPicker = ({
           id={popoverId}
           placement='top'
           data-size='sm'
-          variant={colorError === 'none' ? 'default' : 'warning'}
+          data-color={colorError === 'none' ? 'default' : 'warning'}
```

Comment:
Do we have a `default` color?

Reply:
Maybe it should be `undefined` instead of `default`?

Reply:
> Do we have a `default` color?

Good catch, it probably worked before but now it would not revert to neutral color 🤔 Should be `undefined`. Fixing.